### ISSUE TITLE
Recommend --force-with-lease

### ISSUE THREAD

**Issue by pedrorijo91** [github/training-kit#452]
## Overview
**TL;DR**
Recommend the usage of `git push --force-with-lease` vs the 'traditional' `git push --force`

### Extra Detail

On the [Git Out of Trouble - commit message sucks](https://services.github.com/on-demand/git-trouble/04) section the solution for the *I pushed* scenario references the usage of `git push --force` to solve the situation. We should replace the usage of the `--force` flag with `--force-with-lease`, or at least reference the safer approach of using the latter.

On the other hand, one should think if that's too much information for the targeted user.

#### Reasoning

Depending on the consensus, the implementation seems fairly simple. Possible solutions include (but are not limited to):

* Replace `--force` with `--force-with-lease`
* Add a reference/note to  `--force-with-lease` explaining the differences and recommending its usage

One small paragraph to the *Tell me why* section may also be interesting?

**Comment by crichID** [comment]
@pedrorijo91 this is a great idea. I :heart: your recommendation to replace `--force` with `--force-with-lease` and then add more information in the `Tell me why` section. If someone tries to simply do a push, Git will recommend `push --force` so I think it is important to discuss why we recommend `--force-with-lease` instead, but that should happen in the Tell me why section. 

@pedrorijo91 we would love to have you fork the repo and make this change, or we can do it! Just let us know!
